cgroups/fs/apply_raw.go

```diff
@@ -108,6 +110,28 @@ func (m *Manager) Apply(pid int) error {
 	}
 	m.Paths = paths
 
+	var cpuShares int64
+
+	fd, err := os.Open(path.Join(m.Paths["cpu"], "cpu.shares"))
+	if err != nil {
+		return err
+	}
+
+	_, err = fmt.Fscanf(fd, "%d", &cpuShares)
+	if err != nil && err != io.EOF {
+		return err
+	}
+
+	fd.Close()
+
+	if c.CpuShares != 0 {
+		if c.CpuShares > cpuShares {
+			return fmt.Errorf("The maximum allowed cpu-shares is %d", cpuShares)
+		} else if c.CpuShares < cpuShares {
+			return fmt.Errorf("The minimum allowed cpu-shares is %d", cpuShares)
+		}
+	}
+
```

Comment:
@shishir-a412ed from the CI log, this failed because it needs to import fmt, and declare `c := m.Cgroups`

However, it's probably cleaner to check this after we set the value in `cgroups/fs/cpu.go`